server/src/main/java/org/candlepin/jackson/DateSerializer.java

```diff
@@ -31,20 +31,18 @@
  * The format used is ISO 8601
  */
 public class DateSerializer extends JsonSerializer<Date> {
+    private final SimpleDateFormat dateFormat;
 
-    // This SimpleDateFormat is iso 8601 without milliseconds
-    public static final SimpleDateFormat ISO_8601_WITHOUT_MILLISECONDS;
-
-    static {
-        ISO_8601_WITHOUT_MILLISECONDS = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
-        ISO_8601_WITHOUT_MILLISECONDS.setTimeZone(TimeZone.getTimeZone("UTC"));
+    public DateSerializer() {
+        // This SimpleDateFormat is ISO 8601 without milliseconds
+        this.dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+        this.dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
Fortunately, we don't create this more than once per object mapper, so the per-instance declaration isn't substantially more expensive than the static one. The main reason we made this change, however, is DateFormat (and much of the old Java date bits) is not thread safe. Granted, this doesn't really matter for the same reason changing it to an instance variable doesn't change our performance here, but since the impact is so low, and I was already spending the dev time doing stuff like this, best practices wins out.